executor/join.go

```diff
@@ -785,12 +792,19 @@ type NestedLoopApplyExec struct {
 // Close implements the Executor interface.
 func (e *NestedLoopApplyExec) Close() error {
 	e.innerRows = nil
-
 	e.memTracker = nil
+	if e.runtimeStats != nil {
+		if e.canUseCache {
+			e.runtimeStats.SetCacheInfo(true, e.totalNumber, e.cacheHitNumber)
+		} else {
+			e.runtimeStats.SetCacheInfo(false, e.totalNumber, e.cacheHitNumber)
+		}
```

Comment:
We don't need to call the `SetCacheInfo` since the default value if false.

Reply:
We need this. See `RuntimeStats.SetCacheInfo()` function in `util/execdetails/execdetails.go` file for more details. It can set `e.applyCache = true` in order to explain `applCache` information in `explain analyze`' `execution info` column.